### PR TITLE
R4 - Add patient resource updates

### PR DESCRIPTION
**Pull request by napster235**
Description
----
Add patient resource updates

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

**Comment by napster235** [comment]
Before
Screenshot 2022-09-02 at 15 01 40
After commit a2e1364
Screenshot 2022-09-02 at 15 20 51

**Review by lstefan4 — APPROVED** [review]

File: content/millennium/r4/base/individuals/patient.md [hunk 1]
```diff
@@ -11,23 +11,33 @@ title: Patient | R4 API
...
 
+* The following [HL7® FHIR® US Core Implementation Guide STU 4.0.0](https://hl7.org/fhir/us/core/STU4/){:target="_blank"} Profiles are supported by this resource:
+
+  * [US Core Patient Profile](http://hl7.org/fhir/us/core/STU4/StructureDefinition-us-core-patient.html){:target="_blank"}
+
 The following fields are returned if valued:
 
 * [Patient id](https://hl7.org/fhir/R4/resource-definitions.html#Resource.id){:target="_blank"}
 * [Patient Identifier](https://hl7.org/fhir/R4/patient-definitions.html#Patient.identifier){:target="_blank"}
+  * [Usual](https://hl7.org/fhir/R4/codesystem-identifier-use.html#identifier-use-usual){:target="_blank"}
```
Comment (cerbs) [inline content/millennium/r4/base/individuals/patient.md:22]: Usual isn't an identifier field. Not sure the best way to represent this. Perhaps an implementation note in the read/search sections to indicate that the identifier.use will be 'usual'?

**Review by cerbs — CHANGES_REQUESTED** [review]

**Comment by napster235** [comment]
After commit 6903480
Screenshot 2022-09-07 at 10 59 38
Screenshot 2022-09-07 at 10 59 21

**Review by davischristi — APPROVED** [review]

File: content/millennium/r4/base/individuals/patient.md [hunk 2]
```diff
@@ -51,6 +60,7 @@ Search for Patients that meet supplied query parameters:
 _Implementation Notes_
 
 * Direct secure email will not be returned.
+* Usual will be represented by [Identifier.use](https://hl7.org/fhir/R4/datatypes-definitions.html#Identifier.use){:target="_blank"}
```
Comment (cerbs) [inline content/millennium/r4/base/individuals/patient.md:63]: Looks like I got interrupted yesterday and didn't get this comment posted yesterday. The wording here and line 149 is a bit backward. "Usual" represents the identifier's use, not the other way around.

Perhaps:
```suggestion
* [Patient.identifier.use](https://hl7.org/fhir/R4/datatypes-definitions.html#Identifier.use){:target="_blank"} will have a value of `usual`
```

File: content/millennium/r4/base/individuals/patient.md [hunk 3]
```diff
@@ -136,6 +146,7 @@ List an individual Patient by its id:
...
```
Comment (cerbs) [inline content/millennium/r4/base/individuals/patient.md:149]:
```suggestion
* [Patient.identifier.use](https://hl7.org/fhir/R4/datatypes-definitions.html#Identifier.use){:target="_blank"} will have a value of `usual`
```

**Comment by napster235** [comment]
After commit [3a831af](https://github.com/cerner/fhir.cerner.com/pull/888/commits/3a831aff80c9b74081e0316c31761631a12b3516)
Screenshot 2022-09-09 at 11 49 03
Screenshot 2022-09-09 at 11 49 52

**Review by cerbs — APPROVED** [review]

**Pull request merged**